core-android/src/main/java/com/uber/sdk/android/core/utils/AppProtocol.java

```diff
@@ -26,20 +26,19 @@
 import static com.uber.sdk.android.core.SupportedAppType.UBER_EATS;
 
 public class AppProtocol {
-    @Deprecated
-    public static final String[] UBER_PACKAGE_NAMES =
-            {"com.ubercab", "com.ubercab.presidio.app", "com.ubercab.presidio.exo", "com.ubercab.presidio.development"};
-
     @VisibleForTesting
     static final String[] RIDER_PACKAGE_NAMES =
             {"com.ubercab.presidio.development", "com.ubercab.presidio.exo", "com.ubercab.presidio.app", "com.ubercab"};
     @VisibleForTesting
     static final String[] EATS_PACKAGE_NAMES =
-            {"com.ubercab.eats.debug", "com.ubercab.eats.exo", "com.ubercab.eats"};
```

Comment:
Order is important, go from most developer builds -> production

Reply:
Order does not seem to matter code-wise, but updating regardless.

Notice it doesn't exit early:

```
for (String installedPackage : getSupportedPackageNames().get(selectedApp)) {
    if (PackageManagers.isPackageAvailable(context, installedPackage)) {
        installedPackages.add(
                Pair.create(selectedApp, PackageManagers.getPackageInfo(context, installedPackage)));
    }
}
```

Reply:
My bad, order does matter when looking at `SsoDeeplink#140`. When multiple apps are installed, it will install the first one in the order of this list. We bias towards the most developer-y build.